### PR TITLE
Roboto: Version 3.009; 2024 added

### PR DESCRIPTION
**Pull request by m4rc1e**
4c2ba16: [gftools-packager] Roboto: Version 3.009; 2024 added

* Roboto Version 3.009; 2024 taken from the upstream repo https://github.com/googlefonts/roboto-classic at commit https://github.com/googlefonts/roboto-classic/commit/.

This PR changes the font's license strings to OFL.

cc @garretrieger

### CODE REVIEW

**Comment by m4rc1e** [comment]
@vv-monsalve @emmamarichal Happy to chat about this pr if either of you want to review it.

**Comment by emmamarichal** [comment]
@m4rc1e I'll review it this week, thanks! I'll let you know if I have questions :)

File: ofl/roboto/METADATA.pb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: "Roboto"
 designer: "Christian Robertson, Paratype, Font Bureau"
```
Comment (emmamarichal) [inline ofl/roboto/METADATA.pb:2]: In the previous PR, you removed Paratype and Font Bureau, is it still relevant?

**Comment by emmamarichal** [comment]
@m4rc1e Is there anything more to be done for this PR? From what I remember, the small errors I found can be ignored (ogonek position etc), as they were already there in the previous version.

You can confirm the list of authors in the metadatas, and I'll be able to merge this PR!

File: ofl/roboto/METADATA.pb (hunk 1 above)
Comment (m4rc1e) [reply ofl/roboto/METADATA.pb:2]: We should keep Paratype and Font Bureau since they've both worked on the project.

**Comment by emmamarichal** [comment]
@m4rc1e what is the status here?

**Comment by m4rc1e** [comment]
Still blocked on overlap bit issue

**Pull request merged**